[2023-09-22T16:51:57.443+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:51:57.455+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_age scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2023-09-22T16:51:57.474+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_age scheduled__2021-11-02T00:00:00+00:00 [queued]>
[2023-09-22T16:51:57.476+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 6
[2023-09-22T16:51:57.490+0000] {taskinstance.py:1350} INFO - Executing <Task(_PythonDecoratedOperator): get_age> on 2021-11-02 00:00:00+00:00
[2023-09-22T16:51:57.496+0000] {standard_task_runner.py:57} INFO - Started process 332 to run task
[2023-09-22T16:51:57.503+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_age', 'scheduled__2021-11-02T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/test2.py', '--cfg-path', '/tmp/tmppklkkqdv']
[2023-09-22T16:51:57.508+0000] {standard_task_runner.py:85} INFO - Job 39: Subtask get_age
[2023-09-22T16:51:57.529+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:188 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:51:57.600+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_age scheduled__2021-11-02T00:00:00+00:00 [running]> on host 75cee4990e54
[2023-09-22T16:51:57.619+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:51:58.025+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_age' AIRFLOW_CTX_EXECUTION_DATE='2021-11-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-02T00:00:00+00:00'
[2023-09-22T16:51:58.027+0000] {python.py:183} INFO - Done. Returned value was: 19
[2023-09-22T16:51:58.067+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_age, execution_date=20211102T000000, start_date=20230922T165157, end_date=20230922T165158
[2023-09-22T16:51:58.116+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-09-22T16:51:58.582+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
